Description: Add --skip-validation to xgridfit in Makefile
 This patch is needed to be able to build the fonts.
Author: Gürkan Sengün <gurkan@phys.ethz.ch>
Last-Update: 2011-05-30

--- ttf-junicode-0.6.17+20110525.orig/xgridfit/Makefile
+++ ttf-junicode-0.6.17+20110525/xgridfit/Makefile
@@ -20,19 +20,19 @@ all : regular bold italic bolditalic
 regular : Junicode-Regular.py
 
 Junicode-Regular.py : $(RSCRIPTS)
-	xgridfit -A -m -P no -O Junicode-Regular.py Junicode-Regular.xgf
+	xgridfit --skip-validation -A -m -P no -O Junicode-Regular.py Junicode-Regular.xgf
 
 bold : Junicode-Bold.py
 
 Junicode-Bold.py : $(BSCRIPTS)
-	xgridfit -A -m -P no -O Junicode-Bold.py Junicode-Bold.xgf
+	xgridfit --skip-validation -A -m -P no -O Junicode-Bold.py Junicode-Bold.xgf
 
 italic : Junicode-Italic.py
 
 Junicode-Italic.py : $(ISCRIPTS)
-	xgridfit -A -m -P no -O Junicode-Italic.py Junicode-Italic-Basic.xgf
+	xgridfit --skip-validation -A -m -P no -O Junicode-Italic.py Junicode-Italic-Basic.xgf
 
 bolditalic : Junicode-BoldItalic.py
 
 Junicode-BoldItalic.py : $(BISCRIPTS)
-	xgridfit -A -m -P no -O Junicode-BoldItalic.py Junicode-BoldItalic.xgf
+	xgridfit --skip-validation -A -m -P no -O Junicode-BoldItalic.py Junicode-BoldItalic.xgf
